compiler/env/ProcessorInfo.hpp

```diff
@@ -195,6 +196,50 @@ inline uint32_t getFeatureFlags8Mask()
          | TR_AVX512CD;
    }
 
+enum TR_X86ProcessorFeatures10
```

Comment:
I'd rather start steering these names toward something more descriptive. For example, how about `TR_X86_CPUID_ecx_eax07_ecx00` meaning the result returned from the CPUID instruction in the `ecx` register when `eax` = 7 and `ecx` = 0 .

You may want to add a commit to this PR that changes the other feature flags fields and enums to the new convention.